cfme/tests/containers/test_containers_data_integrity_properties.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from itertools import product
+from cfme.web_ui import CheckboxTable
+from cfme.containers.image import Image
+from cfme.containers.image_registry import ImageRegistry
+from cfme.containers.pod import Pod
+from cfme.containers.provider import Provider
+from cfme.containers.node import Node
+from cfme.containers.container import Container
+from utils import testgen
+from utils.version import current_version
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
```

Comment:
How can I make sure the test knows what is the CheckboxTable I want to work with? How does it find it?